languages/swift/build.sh

```diff
@@ -31,6 +32,8 @@ cat ./tmp/bindings/BitwardenFFI.modulemap ./tmp/bindings/BitwardenCoreFFI.module
 xcodebuild -create-xcframework \
   -library ../../target/aarch64-apple-ios-sim/release/libbitwarden_uniffi.dylib \
   -headers ./tmp/Headers \
+  -library ../../target/aarch64-apple-ios/release/libbitwarden_uniffi.dylib \
+  -headers ./tmp/Headers \
```

Comment:
Do we need to specify -headers twice? It's already there on the previous line.

Reply:
We do. It's headers per library, I guess so you can support different headers for different builds.